platform/android/export/export_plugin.cpp

```diff
@@ -252,6 +252,7 @@ static const char *APK_ASSETS_DIRECTORY = "res://android/build/assets";
 static const char *AAB_ASSETS_DIRECTORY = "res://android/build/assetPacks/installTime/src/main/assets";
 
 static const int DEFAULT_MIN_SDK_VERSION = 21; // Should match the value in 'platform/android/java/app/config.gradle#minSdk'
+static const int VULKAN_MIN_SDK_VERSION = 24;
```

Comment:
Is this the minimum Andriod SDK version? is there a point of still supporting 21-23 through OpenGL with Godot 4 or should we just make 24 the minimum? I'm not sure of the numbers here.

Reply:
We discussed this at length before and agreed that we should support as low as 21 using the OpenGL renderer.